app/controllers/placements/support/schools/mentors_controller.rb

```diff
@@ -4,8 +4,46 @@ class Placements::Support::Schools::MentorsController < Placements::Support::App
 
   def show; end
 
+  def index; end
+
+  def new
+    @mentor_form = mentor_form
```

Comment:
This assignment isn't necessary.
```suggestion
    mentor_form
```